### PR TITLE
doc: Configuring HA NFS server on GlusterFS

### PR DESCRIPTION
**Pull request by soumyakoduri**
Signed-off-by: Soumya Koduri <skoduri@redhat.com>

### CODE REVIEW

File: Administrator Guide/Configuring HA NFS Server.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+## Highly Available Active-Active NFS-Ganesha
+In a highly available active-active environment, if a NFS-Ganesha server that is connected to a NFS client running a particular application crashes, the application/NFS client is seamlessly connected to another NFS-Ganesha server without any administrative intervention.
+The cluster is maintained using Pacemaker and Corosync. Pacemaker acts a resource manager and Corosync provides the communication layer of the cluster.
+Data coherency across the multi-head NFS-Ganesha servers in the cluster is achieved using the UPCALL infrastructure. UPCALL infrastructure is a generic and extensible framework that sends notifications to the respective glusterfs clients (in this case NFS-Ganesha server) in case of any changes detected in the backend filesystem.
+
+### Binaries to be installed
+#### Gluster RPMs (>= 3.7)
+* glusterfs-server
+* glusterfs-ganesha
+#### Ganesha RPMs (>= 2.2)
```
Comment (prashanthpai) [inline Administrator Guide/Configuring HA NFS Server.md]: This part isn't rendered properly.

File: Administrator Guide/Configuring HA NFS Server.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+### Modifying the HA cluster using the `ganesha-ha.sh` script
+Post the cluster creation any further modification can be done using the `ganesha-ha.sh` script. For more information, see the section Modifying the HA cluster using the `ganesha-ha.sh` script. 
+
+## Step-by-step guide
+### Configuring NFS-Ganesha using Gluster CLI⁠
+#### Pre-requisites to run NFS-Ganesha
+Ensure that the following pre-requisites are taken into consideration before you run NFS-Ganesha in your environment:
+
+ * A Red Hat Gluster Storage volume must be available for export and NFS-Ganesha rpms are installed.
```
Comment (prashanthpai) [inline Administrator Guide/Configuring HA NFS Server.md]: No downstream product reference please

File: Administrator Guide/Configuring HA NFS Server.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+    where,
+    HA_CONF_DIR: The directory path containing the ganesha-ha.conf file.
+    HOSTNAME: Hostname of the new node to be added
+#### Modifying the default export configuration
+To modify the default export configurations perform the following steps on any of the nodes in the existing ganesha cluster:
+*   Edit/add the required fields in the corresponding export file located at `/etc/ganesha/exports`.
+*   Execute the following command:
+
+    -#./ganesha-ha.sh --refresh-config <HA_CONFDIR> <volname>
```
Comment (prashanthpai) [inline Administrator Guide/Configuring HA NFS Server.md]: This is incorrectly indented, hence it's not rendered as intended to be.

File: Administrator Guide/Configuring HA NFS Server.md (hunk 1 above)
Comment (soumyakoduri) [inline Administrator Guide/Configuring HA NFS Server.md]: Done.

File: Administrator Guide/Configuring HA NFS Server.md (hunk 2 above)
Comment (soumyakoduri) [inline Administrator Guide/Configuring HA NFS Server.md]: Done.

File: Administrator Guide/Configuring HA NFS Server.md (hunk 3 above)
Comment (soumyakoduri) [inline Administrator Guide/Configuring HA NFS Server.md]: Done.

**Comment by humblec** [comment]
@soumyakoduri can you please make an entry in "mkdocs.yml" for this new file , then only it will properly displayed in gluster.readthedocs.

**Comment by soumyakoduri** [comment]
Done.

**Pull request merged**